### PR TITLE
Update Chinese Simplified translation for README

### PR DESCRIPTION
**Pull request by yume-chan**

### CODE REVIEW

**Comment by rom1v** [comment]
Thank you :+1:

@Win7GM, as the author of #1723, could you review, please?

**Comment by Win7GM** [comment]
Sure. Will find time to do it.

**Comment by Win7GM** [comment]
> Thank you 👍
> 
> @Win7GM, as the author of #1723, could you review, please?

Sure. Will find time to do it.

File: README.zh-Hans.md [hunk 1]
```diff
@@ -2,271 +2,289 @@ _Only the original [README](README.md) is guaranteed to be up-to-date._
...
- - **质量** （分辨率可达1920x1080或更高）
- - **低延迟** (35-70ms)
- - **快速启动** （数秒内即能开始显示）
- - **无侵入性** （不需要在安卓设备上安装任何程序）
+ - **轻量** (原生，仅显示设备屏幕)
+ - **性能** (30~60fps)
+ - **质量** (分辨率可达 1920×1080 或更高)
+ - **低延迟** ([35~70ms][lowlatency])
+ - **快速启动** (最快 1 秒内即可显示第一帧)
+ - **无侵入性** (不会在设备上遗留任何程序)
```
Comment (Win7GM) [inline README.zh-Hans.md:22]: 稍微有点好奇，为什么要用英文的括号？还有上面那段也是

Comment (yume-chan) [reply README.zh-Hans.md:22]: 因为全角括号接其它全角符号（比如右括号加句号“）。”）或者接半角英数字符的时候间距会过大，所以一些本地化规范会推荐全部使用半角括号，然后用空格来控制额外的间距。比如虽然不完全一样，微软的建议中文本地化规范就要求使用半角括号。

Comment (rom1v) [reply README.zh-Hans.md:22]: @Win7GM Is it ok to merge? (have you finished the review?)

Comment (Win7GM) [reply README.zh-Hans.md:22]: > @Win7GM Is it ok to merge? (have you finished the review?)

Yes it's ok. 👍

**Comment by rom1v** [comment]
Thank you, merged into `master`: ce43fad645d4eb30f322dbeb50d5197601564931

**Pull request closed without merging**